src/SUMMARY.md

```diff
@@ -1,8 +1,8 @@
 
 ## The Big Picture
 
-* [How we think](overview/how-we-think.md)
-* [How it looks](overview/how-it-looks.md)
+* [How we think](overview/basics.md)
+* [How it looks](overview/structure.md)
```

Comment:
Because previous titles lack clarity of purpose. The section called "How it looks" does not concern the looks of the thing. _sometimes_ I prefer clarity to cuteness.